QC/RawCompressed/noise_analysis.C

```diff
@@ -0,0 +1,137 @@
+#include "TOFWorkflowUtils/CompressedAnalysis.h"
```

Comment:
Ciao Francesca, questa non c'e' gia' in https://github.com/alicetof/Commissioning/blob/master/ANALYSIS/noise/noise_analysis.C ? Non puoi usare questa nel .sh?